hibernate/hibernate-reactive/src/main/java/io/quarkus/ts/hibernate/reactive/http/ValidationResource.java

```diff
@@ -48,6 +50,28 @@ public Uni<Response> createPerson(String name) {
                 });
     }
 
+    @PUT
+    @Path("/xml/customer/{name}/{email}")
+    public Uni<Response> createXmlCustomer(@PathParam("name") String name, @PathParam("email") String email) {
+        XmlValidatedCustomer customer = new XmlValidatedCustomer(name, email);
+
+        return sessionFactory.withTransaction(session -> session.persist(customer))
+                .map(nothing -> Response.status(Response.Status.CREATED).build())
+                .onFailure()
+                .recoverWithItem(e -> {
+                    Throwable cause = e;
+                    while (cause != null) {
```

Comment:
nitpick (but really nitpick :grin: ) this should be do while considering you know the that `e` is not `null`